backend/src/backend/main.py

```diff
@@ -137,4 +137,18 @@ async def add_book(book_payload: BookAddSchema):
     return {"msg": "ok", "uuid": str(book.id), "title": book.title.ident}
 
 
+@database.transaction()
+@api.get("/titles/", status_code=200, response_model=TitlesListSendSchema)
+async def receive_titles():
+    titles = await Title.objects.select_all().all()
```

Comment:
We should explicitly select what we want to return here (`ident`) in order to request just what's required from the DB. Titles are pretty thin at the moment but might evolve.